h2/src/main/org/h2/mvstore/MVMap.java

```diff
@@ -1179,6 +1326,10 @@ public final String toString() {
          * Counter for attempted root updates.
          */
         public  final    long          updateAttemptCounter;
+        /**
+         * Size of the occupied part of the append buffer.
+         */
+        public  final    byte          appendCounter;
```

Comment:
How exactly this field is protected from overflow?

Reply:
By check in append() :
```if (rootReference.appendCounter >= keysPerPage) {```

Reply:
127? Unfortunately, I don't know typical values of `keysPerPage`, may be it's better to document it somewhere.